utils_cv/detection/plot.py

```diff
@@ -124,6 +150,44 @@ def display_bboxes(
         plt.show()
 
 
+def display_bbox_mask(
```

Comment:
Should be called "display_bboxes_mask", similar to what it was called before since it takes multiple bboxes as input

Reply:
Renamed